docs/getting_started/faq.rst

```diff
@@ -0,0 +1,110 @@
+Frequently Asked Questions (FAQs)
+=================================
+
+Below, you will find answers to the most commonly asked questions about
+Modin. If you still cannot find the answer you are looking for, please post on 
+the #support channel on our Slack_ community or open a Github issue_.
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+In pandas, you are only able to use one core at a time when you are doing computation of
```

Comment:
```suggestion
While pandas work extremely well on small datasets, as soon as you start working with 
medium to large datasets that are more than a few GBs, pandas can become painfully 
slow or run out of memory. This is because pandas is single-threaded, in other words, 
you can only process your data with one core at a time.
```